readme.md

```diff
@@ -61,11 +61,8 @@ Integration tests can be run from the integrationtest module and the default beh
 unpack the modules in it and run the tests using Arquillian.
 
 
-[WARNING]
-====
-Be careful when using on existing installation since the modules used by the build are going to be extracted into the
-server you want to run the test, changing the original setup.
-====
+# [WARNING]
```

Comment:
Looks like we mixed asciidoctor and markdown syntaxes.
I guess I'd prefer Warning to be a subsection i.e. `#### Warning` and leave the rest of the warning paragraph unchanged.